apps/web/components/FrontpageSlider/FrontpageSlider.treat.ts

```diff
@@ -62,6 +62,7 @@ export const textItemVisible = style({
 })
 
 export const tabBullet = style({
+  display: 'inline-block',
```

Comment:
Was this implicit before?
Does this rule change behaviour in any way?

Reply:
The element using the class was a div but changed to span and needed to be block/inline-block

Reply:
:+1: